C++/C++.sublime-syntax

```diff
@@ -52,7 +52,7 @@ variables:
   visibility_modifiers: 'private|protected|public'
   other_keywords: 'typedef|nullptr|{{visibility_modifiers}}|static_assert|sizeof|using|typeid|alignof|alignas|namespace|template'
   modifiers: '{{storage_classes}}|{{type_qualifier}}|{{compiler_directive}}'
-  non_angle_brackets: '(?=<<|<=)'
```

Comment:
I'm not certain this change is necessary, but it looks like this pattern already includes `<=` which shares the same starting tokens. I can't think of a scenario that would need to match `=<<` though...?